src/Psalm/Internal/Cli/Psalter.php

```diff
@@ -354,7 +364,11 @@ public static function run(array $argv): void
             );
 
             if ($doc_block_add_new_line_before_return === null) {
-                die('--add-newline-between-docblock-annotations expects a boolean value [true|false|1|0]' . PHP_EOL);
```

Comment:
We should also add `die` to `psalm.xml/forbiddenFunctions`, so that future code consistently uses `exit` instead.

Reply:
Just `psalm.xml.dist`. Even though `die()` and `exit()` are synonymous, I have the impression that people generally use `die()` with a string argument and `exit()` with an int, and we want Psalm sources to use the latter.